libs/application/templates/parental-leave/src/lib/ParentalLeaveTemplate.ts

```diff
@@ -728,13 +742,27 @@ const ParentalLeaveTemplate: ApplicationTemplate<
           ],
           [DefaultEvents.ABORT]: [
             {
+              cond: (application) => goToState(application, States.APPROVED),
               target: States.APPROVED,
             },
+            {
+              cond: (application) =>
+                goToState(application, States.VINNUMALASTOFNUN_APPROVAL),
+              target: States.VINNUMALASTOFNUN_APPROVAL,
+            },
+            {
+              cond: (application) =>
+                goToState(application, States.VINNUMALASTOFNUN_APPROVE_EDITS),
+              target: States.VINNUMALASTOFNUN_APPROVE_EDITS,
+            },
           ],
```

Comment:
Maybe we need a default target here.
For example if all condition is false then ABORT doesn't go anywhere. Let's 'vinnumalastofnun_approve_edits' be the defaults one

Reply:
Ok 👍